tools/workflow/cleaner/lambda/clean.go

```diff
@@ -79,3 +154,19 @@ func shouldDelete(lf *lambda.FunctionConfiguration) (bool, error) {
 	}
 	return false, nil
 }
+
+func shouldDeleteLayer(layerList *lambda.LayersListItem) *string {
+	layerARN := layerList.LayerArn
+	regexList := []string{
+		".*:layer:[a-zA-Z0-9-_]+.*$",
+	}
```

Comment:
yeah, i agree. Currently we have many more unused test layers with various naming conventions and its vast and i think they are consuming a good chunk of it. So we can take an approach of clearing all of them existing and then take follow up to make it more restricitve (matching with current layer naming conventions). if that sounds good.

Reply:
No, I would prefer to use more restrictive regexs from the start. Don't forgot this will run in all regions. I took a look through the lambda layer names and there are some pretty predictable lambda layer names there. Please make a best effort of creating a list of regexs for the most common ones. If we miss some after the first PR that's okay, we can always expand the regex list.

Reply:
I agree

Reply:
You could also probably do some fancy shell scripting to figure out all the unique prefixes. Then just use that to build your regex list.

Reply:
I have added regexes for 9 most common, just changed the keyword to be specific and its seemed to have worked